local/scripts/dist.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
This is where we take the release bundle and push it to the `stream-dist` repository. We always push the branch and add a tag if this is a tag build on travis as indicated by `$TRAVIS_TAG`.